### PR TITLE
New package: pgadmin4-4.26

### PR DESCRIPTION
**Pull request by mustaqimM**
I haven't as yet used it extensively but it seems to work. The executable is used only to launch the python backend; should I rename the package to just: `pgadmin4`?

### CODE REVIEW

**Comment by Chocimier** [comment]
- Yes, name packages after tarball (or top dir in case no name in tarball), including letter case, except of flask-* where is good to have all titlecased.
- remove python2 packages
- resolve xlint warnings

then will review further.

**Comment by mustaqimM** [comment]
Do you mean I should not compile for python2? (I actually had it like that and later decided to add it after all)

File: srcpkgs/python-brotli/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+version=1.0.7
+revision=1
+wrksrc="Brotli-${version}"
+build_style="python-module"
+hostmakedepends="python-setuptools python-devel python3-setuptools python3-devel"
+short_desc="Python3 brotli module"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="Apache-2.0"
+homepage="https://pypi.org/project/Brotli/"
+distfiles="https://files.pythonhosted.org/packages/cd/9c/7955895f5672ecc85270244582c6b53ff95bb4c24bf77bd9271d42351635/Brotli-1.0.7.zip"
```
Comment (sgn) [inline srcpkgs/python-brotli/template:12]:
```sh
distfiles=$PYPI_SITE/B/Brotli/Brotli-${version}.tar.gz
```

Comment (sgn) [reply srcpkgs/python-brotli/template:12]: Drop Python 2, too. It's EOL

Comment (mustaqimM) [reply srcpkgs/python-brotli/template]: The brotli package isn't under the pypi site, I changed it now but I forgot to rebuild, it won't work with the new url

Comment (sgn) [reply srcpkgs/python-brotli/template]: OK, it should be:
```sh
distfiles=$PYPI_SITE/B/Brotli/Brotli-${version}.zip
```

File: srcpkgs/python3-Flask-Security-Too/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+version=3.4.3
+revision=1
+wrksrc="flask-security-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-devel"
+short_desc="Simple security for Flask apps"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="MIT"
+homepage="https://github.com/Flask-Middleware/flask-security/"
+distfiles="https://github.com/Flask-Middleware/flask-security/archive/${version}.tar.gz"
```
Comment (sgn) [inline srcpkgs/python3-Flask-Security-Too/template:12]: I prefer
```
distfiles="$PYPI_SITE/F/Flask-Security/Flask-Security-${version}.tar.gz"
checksum=d61daa5f5a48f89f30f50555872bdf581b2c65804668b0313345cd7beff26432
```

Same for other packages.

**Pull request closed without merging**